CHANGELOG.md

```diff
@@ -17,6 +17,11 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.64.12] - 2023-11-12
+### Added
+- Added Scope `DataSet` for `WorkflowOrchestrationAcl`.
+- Added optional `data_set_id` to `Workflow` and `WorkflowUpsert`.
```

Comment:
Just a suggestion, I do like how precise you are with the added features, but the overall important message is imo that workflows now support data sets:
```suggestion
## [7.64.12] - 2023-11-14
### Added
- Workflows now support data sets
```

Reply:
Fair enough!